src/main/settings.py

```diff
@@ -97,6 +97,8 @@
     },
 ]
 
+TEMPLATES[0]["DIRS"].append(os.path.join(BASE_DIR, "tests", "templates"))
```

Comment:
Why don't we edit the existing `TEMPLATES` settings instead?

Reply:
I deleted the line and modified TEMPLATES